HelpSource/Classes/ArrayedCollection.schelp

```diff
@@ -172,13 +197,21 @@ method::plot
 Plot values in a GUI window.  See link::Reference/plot:: for more details. When the receiver contains code::nil:: items, the plot fails with an error.
 
 method::swap
-Swap the values at indices i and j.
+Swap the values at indices i and j, both specified as integers starting from 0.
 code::
 [1, 2, 3].swap(0, 2).postln;
 ::
 
 method::put
-Put strong::item:: at strong::index::, replacing what is there.
+Put strong::item:: at strong::index:: starting from 0, specified as an integer or float, or even array of float, replacing what is there.
```

Comment:
I'd recommend de-emphasizing use of floats:
```
Put strong::item:: at strong::index:: starting from 0, replacing what is there. strong::index:: can be an array of indices. If a float is used as index, it will be truncated.
```
or similar.
This same approach could be used in other places where this point is repeated.

Reply:
Thanks! I searched the text `array of float` and could not find anything. 

Done in https://github.com/supercollider/supercollider/pull/6536/commits/b851561fdd38e8719f856ef62b107f48689e2da5.